ai-service/src/ai_service/db.py

```
@@ -6,17 +6,19 @@
 import chromadb
 import os
 from dotenv import load_dotenv
+from .exceptions import DatabaseError, NotFound
 
 load_dotenv()
 
 # Set up ChromaDB client and persistent collection
 chroma_path = os.getenv("CHROMA_STORE_PATH")
 if not chroma_path:
-    raise ValueError("CHROMA_STORE_PATH environment variable is required")
+    raise NotFound("CHROMA_STORE_PATH environment variable is required")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)